certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4627 @@
+{
+  "x-generator": "NSwag v13.14.8.0 (NJsonSchema v10.6.1.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "AIForged",
+    "description": "Integration with AI Forged system",
```

Comment:
@aiforged Please go thru all the instructions mentioned in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and provide a detailed description which everyone would understand about your service/product.